File: src/domain/value_objects/wish_item_status.rs
#![allow(dead_code)]
/// 欲しいものの状態
/// 値オブジェクト（ステータス列挙）
#[derive(Debug, Clone, PartialEq, Eq)]
pub enum WishItemStatus {
    Inbox,
    NextToBuy,
    OnHold,
    Archived,
    Purchased,
}

File: frontend/src/utils/wishItemStatus.ts
import type { WishItemStatus } from '../api/types'

export const STATUS_LABELS: Record<WishItemStatus, string> = {
  Inbox: '検討中',
  NextToBuy: '次に買う',
  OnHold: '保留中',
  Archived: '見送り',
  Purchased: '購入済み',
}
